pyinstaller/electron/main.js

```diff
@@ -70,6 +64,11 @@ if (isMac && isDev) {
   app.dock.setIcon(dockIcon)
 }
 
+let appSettings = getAppSettings()
+let dimensions = { widIth: 1500, height: 1000 };
```

Comment:
Typo on width here @moneymanolis

Reply:
You can use the "Add a suggestion" functionality for this little stuff.